packages/x-date-pickers/src/CalendarPickerSkeleton/CalendarPickerSkeleton.tsx

```diff
@@ -56,7 +56,7 @@ const CalendarPickerSkeletonWeek = styled('div', {
 
 const CalendarPickerSkeletonDay = styled(Skeleton, {
   name: 'MuiCalendarPickerSkeleton',
-  slot: 'Day',
+  slot: 'DaySkeleton',
```

Comment:
Align naming with className

Reply:
Nope, it's just a consistency in `className` and `slot` fields.
I'm not 100% sure what does this slot name do, but the applied class name depends on how we generate them using `getCalendarPickerSkeletonUtilityClass`